app/main.py

```diff
@@ -1 +1,14 @@
-# write your code here
+def copy_file(command: str) -> None:
+    split_command = command.split()
+    cmd, input_path, output_path = split_command
```

Comment:
thus line must be after if len(split_command) != 3: because if split_command will have less then 3 arguments code will be broken